sys/arm64/include/atomic.h

```diff
@@ -667,21 +667,19 @@ static __inline void							\
 atomic_##op##_##bar##ptr(volatile uintptr_t *p, uintptr_t val)		\
 {									\
 	uintptr_t previous;						\
-	ptraddr_t tmp1, tmp2;						\
+	ptraddr_t tmp;							\
 	int res;							\
 									\
 	__asm __volatile(						\
 		"1:"							\
 		"ld" #a "xr %0, [%4]\n"					\
 		"gcvalue %2, %0\n"					\
```

Comment:
This could be eliminated if there was an equivalent to `%wN` to denote the X version of the C register as then we could use that as the second operand in the `scvalue` instruction below.